department-of-veterans-affairs/va.gov-cms
# VACMS-3910: Add details with image fieldgroup formatter type.

## Implementation notes
Once released to prod, will require entering the svg assets found here: https://drive.google.com/drive/u/1/folders/1wX2Ay1F8cLzRPpmrRSA3dltC4Yce_Vi4 with corresponding alt text for each svg, found here: https://www.figma.com/file/VFo6GLTHXwuyNeNCSdl2YU/CLP-visual-guides?node-id=0%3A1 on the clp form found here: /admin/structure/types/manage/campaign_landing_page/form-display

## Description
See #3910 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/105517352-f0d7d280-5ca4-11eb-92fd-3f86aa40493d.png)
![image](https://user-images.githubusercontent.com/2404547/105517363-f503f000-5ca4-11eb-8962-ced4222b33eb.png)

## QA steps
- As an administrator, go to admin/structure/types/manage/campaign_landing_page/form-display and open up the Hero panel.
- Visually verify that it reads `Details with image`
- Add an svg from assets folder: https://drive.google.com/drive/u/1/folders/1wX2Ay1F8cLzRPpmrRSA3dltC4Yce_Vi4 to this panel, and add some alt text.
- Repeat this process for the `Why this matters` panel
- Save, then visit `node/15051/edit`
- Visually verify that the svgs appear at the top of the expanded fieldsets, and the images have the alt text you entered on the clp form.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
